builtin/providers/azurerm/resource_arm_network_interface_card.go

```diff
@@ -252,6 +258,17 @@ func resourceArmNetworkInterfaceRead(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
+	if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
+		var privateIPAddress *string
+		for _, ipConfig := range *iface.IPConfigurations {
+			privateIPAddress = ipConfig.Properties.PrivateIPAddress
```

Comment:
At the moment we can only specify one ip configuration.
https://github.com/Azure/azure-sdk-for-go/issues/259

Maybe we can pick the primary ip configuration once #259 is done.